applications/h264/h264_endoscopy_tool_tracking/python/h264_endoscopy_tool_tracking.yaml

```diff
@@ -0,0 +1,141 @@
+%YAML 1.2
+# SPDX-FileCopyrightText: Copyright (c) 2022-2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+extensions:
+  - libgxf_cuda.so
+  - libgxf_multimedia.so
```

Comment:
These GXF extensions are loaded by default since at least v0.6 so don't need to be listed here
```suggestion
```